build.py

```diff
@@ -249,6 +249,8 @@ def unpack(self):
                 zf.extractall(path=destdir)
         # .. and set the builder object to point to the file
         self.builder.ninja = destfile

```

Comment:
We need to launch directly ninja,exe for the build process, like nuget, The path is needed for meson so it finds ninja and don't complain. We can use only one (the path?) and calculate the other.